src/pages/rechner/[frage].tsx

```diff
@@ -3,16 +3,16 @@ import React, { useEffect, useState } from 'react';
 import { InputField } from '../../elements/input-field';
 import { Copy } from '../../identity/copy';
 import { Heading2 } from '../../identity/heading-2';
-import { Question } from '../../data/questions';
-import Questions from '../../data/questions.json';
 import { Heading1 } from '../../identity/heading-1';
 import { getLocalStorage, setLocalStorage } from '../../utils/local-storage';
 import { LinkElement } from '../../elements/link';
 import { useRouter } from 'next/dist/client/router';
 import { CategoriesNavigation } from '../../compositions/sidebar';
+import { initializeFirestore } from '../../utils/get-firestore';
+import { DocumentData } from '@firebase/firestore/dist/lite';
 
 type Props = {
-  question: Question;
+  question: DocumentData;
```

Comment:
`DocumentData` is a generic type from firestore which doesn't help you at all to know which properties are available on your question object. Please make sure you properly type the data you get from firestore.